public/js/index.js

```diff
@@ -1,40 +1,44 @@
 'use strict';
 
+var app = app || {};
 
-(function () {
+
+(function (module) {
 
   function hamburgerClick() {
     $('#nav img').fadeIn();
     $('#hamburger').hide();
   }
 
-  function aboutClick() {
-    $('#listOfProjects').hide();
-    $('#aboutMeContent').fadeIn();
-  }
-
   $('#hamburger').on('click', hamburgerClick);
 
-  $('#aboutMeButton').on('click', aboutClick);
-
-  var rawHtml = $('#handlebars').html();
-  var templateFunction = Handlebars.compile(rawHtml);
-
   function Project (obj) {
     this.name = obj.projName;
     this.location = obj.projLink;
   }
 
+  Project.aboutClick = () => {
+    $('#listOfProjects').hide();
+    $('#aboutMeContent').fadeIn();
+  }
+
+  Project.homeClick = () => {
+    $('#aboutMeContent').hide();
+    $('#listOfProjects').fadeIn();
+  }
```

Comment:
These click methods should be in a controller file rather than in with the Project model code to be consistent with MVC organization.